src/index.html

```diff
@@ -60,41 +60,64 @@ <h1 class='f3 fw2 montserrat aqua ttu ma0'>Public Gateways</h1>
-          We will limit collection of metrics to only necessary features, 'sessions' and 'views'. See <a href="https://support.count.ly/hc/en-us/articles/360037441932-Web-analytics-JavaScript-#features-for-consent">Countly's group_features</a> for more information.
+
+    <div class="hidden js-metrics-notification-modal metrics-notification-wrapper bg-navy bt bw1 border-jade">
+      <div class="metrics-modal-close js-modal-close">✕&nbsp</div>
+      <div class="metrics-notification-content js-metrics-agreement">
+        <h3 class="js-metrics-notification-heading metrics-notification-heading montserrat">
+          Metrics and Telemetry Notice
+        </h3>
+        <!-- metric agreement -->
+        <span class="metrics-notification-text montserrat">
+          This site collects metrics and other telemetry data in accordance with our <a href="https://github.com/ipfs/ipfs-gui/issues/125">metrics collection policy</a>. You can learn more about
+          how we use this information and customize your preferences by clicking “manage settings” below.
         </span>
         <div class="metrics-notification-buttons">
-          <button id="metrics-notification-warning-close" class="js-metrics-notification-warning-close">✕&nbsp;Close</button>
+          <button id="metrics-notification-manage" class="js-metrics-notification-manage">Manage settings</button>
+          <button id="metrics-notification-confirm" class="primary js-metrics-notification-confirm">Got it!</button>
         </div>
       </div>
-    </div>
-    <div class="hidden js-metrics-notification metrics-notification-wrapper bg-navy bt bw1 border-aqua">
-      <div class="metrics-notification-spacer"></div>
-      <div class="metrics-notification-container">
-        <span class="metrics-notification-text">We're collecting <a href="https://github.com/ipfs/ipfs-gui/issues/125">web-vitals, pageview, and other metrics</a> in order to improve and prioritize our work on IPFS and its public gateways.
-          Please consent to the collection of these metrics to assist in our efforts!</span>
+      <div class="hidden metrics-notification-content metrics-notification-content-preferences js-metrics-preferences">
+        <h3 class="js-metrics-notification-heading metrics-notification-heading montserrat">
+          Metrics and Telemetry Settings
+        </h3>
+        <!-- manage preferences -->
+        <span class="metrics-notification-text montserrat">
+          This site collects metrics and other telemetry data in order to improve and prioritize work on IPFS. It does not collect any personally identifiable information.
+        </span>
+        <div class="metrics-notification-customize">
+          <span class="metrics-notification-text metrics-notification-customize-text montserrat">
+            Opt-in or out at any time by using the toggle below.
+            For more details, read the full<a href="https://github.com/ipfs/ipfs-gui/issues/125"> metrics collection policy</a>.
```

Comment:
we should link to https://github.com/ipfs-shipyard/ignite-metrics/blob/main/docs/telemetry/COLLECTION_POLICY.md

```suggestion
            For more details, read the full<a href="https://github.com/ipfs-shipyard/ignite-metrics/blob/main/docs/telemetry/COLLECTION_POLICY.md"> metrics collection policy</a>.
```